src/v2/providers/alerts/crashlytics.ts

```diff
@@ -0,0 +1,325 @@
+  return onOperation<StabilityDigestPayload>(
+    stabilityDigestAlert,
+    appIdOrOptsOrHandler,
+    handler
+  );
+}
+
+/** Handle a velocity alert published */
+export function onVelocityAlertPublished(
+  handler: (event: CrashlyticsEvent<VelocityAlertPayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<VelocityAlertPayload>>;
+export function onVelocityAlertPublished(
+  appId: string,
+  handler: (event: CrashlyticsEvent<VelocityAlertPayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<VelocityAlertPayload>>;
+export function onVelocityAlertPublished(
+  opts: CrashlyticsOptions,
+  handler: (event: CrashlyticsEvent<VelocityAlertPayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<VelocityAlertPayload>>;
+export function onVelocityAlertPublished(
+  appIdOrOptsOrHandler:
+    | string
+    | CrashlyticsOptions
+    | ((event: CrashlyticsEvent<VelocityAlertPayload>) => any | Promise<any>),
+  handler?: (
+    event: CrashlyticsEvent<VelocityAlertPayload>
+  ) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<VelocityAlertPayload>> {
+  return onOperation<VelocityAlertPayload>(
+    velocityAlert,
+    appIdOrOptsOrHandler,
+    handler
+  );
+}
+
+/** Handle a new ANR issue published */
+export function onNewAnrIssuePublished(
+  handler: (event: CrashlyticsEvent<NewAnrIssuePayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<NewAnrIssuePayload>>;
+export function onNewAnrIssuePublished(
+  appId: string,
+  handler: (event: CrashlyticsEvent<NewAnrIssuePayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<NewAnrIssuePayload>>;
+export function onNewAnrIssuePublished(
+  opts: CrashlyticsOptions,
+  handler: (event: CrashlyticsEvent<NewAnrIssuePayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<NewAnrIssuePayload>>;
+export function onNewAnrIssuePublished(
+  appIdOrOptsOrHandler:
+    | string
+    | CrashlyticsOptions
+    | ((event: CrashlyticsEvent<NewAnrIssuePayload>) => any | Promise<any>),
+  handler?: (event: CrashlyticsEvent<NewAnrIssuePayload>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<NewAnrIssuePayload>> {
+  return onOperation<NewAnrIssuePayload>(
+    newAnrIssueAlert,
+    appIdOrOptsOrHandler,
+    handler
+  );
+}
+
+/** @internal */
+export function onOperation<T>(
+  alertType: string,
+  appIdOrOptsOrHandler:
+    | string
+    | CrashlyticsOptions
+    | ((event: CrashlyticsEvent<T>) => any | Promise<any>),
+  handler: (event: CrashlyticsEvent<T>) => any | Promise<any>
+): CloudFunction<FirebaseAlertData<T>> {
+  if (typeof appIdOrOptsOrHandler === 'function') {
+    handler = appIdOrOptsOrHandler as (
+      event: CrashlyticsEvent<T>
+    ) => any | Promise<any>;
+    appIdOrOptsOrHandler = {};
+  }
+
+  const [opts, appId] = getOptsAndApp(
+    appIdOrOptsOrHandler as string | CrashlyticsOptions
+  );
+
+  const func = (raw: CloudEvent<unknown>) => {
+    return handler(raw as CrashlyticsEvent<T>);
```

Comment:
Here's where you might need to recognize some date fields and change their format (if API council decides that ISO 8601 isn't the right format). I know you've already gone through review, but it just might be worth bringing up to a few people (e.g. @mbleigh) that we have inconsistencies and need an AIP.